extras/i2c/i2c.h

```diff
@@ -41,9 +41,11 @@ extern "C" {
  * Define i2c bus max number
  */
 #ifndef MAX_I2C_BUS
-    #define MAX_I2C_BUS 2
+    #define I2C_MAX_BUS 2
```

Comment:
Would want to change the `#ifndef MAX_I2C_BUS` too here.

Also this patch will need to be edited to remove the lwip changes. If working with modules is a challenge then just save the diffs to a file and remove all the changes from this branch then re-apply and make a forced push. Then just don't commit the lwip directory.

Reply:
Thank you for your review!